docs/release-notes/8.4.0-1.md

```diff
@@ -1,2 +1,41 @@
 #  Percona XtraBackup 8.4.0-1 (2024-0-)
 
+Get started with [Quickstart Guide for Percona XtraBackup].
+
+## Release highlights
+
+Percona XtraBackup 8.4.0-1 is based on MySQL 8.4 Long Term Supported (LTS) Release. This release allows taking backups of MySQL 8.4 and MySQL 8.4-compatible servers.
```

Comment:
Do you think we should clarify about taking backups of 8.4.1 and 8.4.2 servers here?
Or we will do this else where? @patrickbirch ^

Reply:
updated